worlds/smz3/Options.py

```diff
@@ -0,0 +1,35 @@
+import typing
+from Options import Choice, Option
+
+class SMLogic(Choice):
+    displayname = "SMLogic"
+    option_Normal = 0
+    option_Hard = 1
+    default = 0
+
+class SwordLocation(Choice):
+    displayname = "Sword Location"
+    option_Randomized = 0
+    option_Early = 1
+    option_Uncle = 2
+    default = 0
+
+class MorphLocation(Choice):
+    displayname = "Morph Location"
```

Comment:
the attribute for display name was changed to `display_name`, to comply with pep8